reconcile/change_owners/decision.py

```diff
@@ -1,7 +1,10 @@
 from collections import defaultdict
 from dataclasses import dataclass
 from enum import Enum
-from typing import Any
+from typing import (
+    Any,
+    Iterable,
```

Comment:
My understanding is we should from now on import that from `collections.abc` for [PEP-585](https://peps.python.org/pep-0585/) compliance.

Reply:
:facepalm: you are right. thanks!!

Reply:
fixed